core/model-vocabulary/src/main/java/org/eclipse/rdf4j/model/vocabulary/CONFIG.java

```diff
@@ -20,6 +21,7 @@
  *
  * @since 4.3.0
  */
+@Experimental
 public class CONFIG {
```

Comment:
And I marked the CONFIG class as experimental so that we can make changes in 5.0.0 if we want. The idea is to allow users to test out the new CONFIG code in 4.3.0 and provide us with feedback. Then we still have the option to integrate that feedback into the code for the 5.0.0 release.